data/test_config/config_spacy_entity_extractor.yml

```diff
@@ -1,6 +1,7 @@
 language: en
 pipeline:
     - name: "SpacyNLP"
+      model: "en_core_web_md"
```

Comment:
Do we also need to make this change to all the example bots?

Reply:
If they use spaCy, yes. I'm reminded that this would also include all examples in the docs. Note that we're not introducing the `model` variable, we're merely requiring that it's filled in explicitly. 

Just to check, @joejuzl what example bots were you thinking about? Our starter packs?

Reply:
I think Joe means the one in https://github.com/RasaHQ/rasa/tree/main/examples

Reply:
Done. Only `moodbot` and `reminderbot` used spaCy.